tests/components/automatic/test_device_tracker.py

```diff
@@ -108,12 +103,6 @@ def get_trips(*args, **kwargs):
     session.get_trips.side_effect = get_trips
     session.refresh_token = "mock_refresh_token"
 
-    @asyncio.coroutine
-    def ws_connect():
-        return asyncio.Future()
-
-    mock_ws_connect.side_effect = ws_connect
```

Comment:
Don't we still need to patch this though?  We're not using `asynctest` which would automatically mock async methods with awaitables, this was patched with plain old `unittest`.  I suggest importing `asynctest` instead of `unittest` in this module.  It should be backwards compatible.

Reply:
Done, added an additional assert to show it is being called.